Dockerfile

```diff
@@ -20,6 +20,7 @@ COPY Makefile .
 
 COPY go.mod .
 COPY go.sum .
+COPY ./api /go/api
```

Comment:
Can we try doing `ADD api api` on L13 instead?

Reply:
Attemped to move this to `ADD`, looks like docker image built successfully. But we are seeing this sonar test failure on all PR workflows at the moment, I think it is outside of our control, assuming somebody somewhere will fix